src/main/java/org/team199/robot2022/subsystems/Shooter.java

```diff
@@ -64,10 +63,6 @@ public Shooter() {
         slave.setIdleMode(IdleMode.kCoast);
         
         SmartDashboard.putNumber("Ball PSI", ballPSI);
-
-        Log.registerDoubleVar("Shooter RPM", () -> pidController.getEncoder().getVelocity());
-        Log.registerDoubleVar("Shooter Current Master", () -> master.getOutputCurrent());
-        Log.registerDoubleVar("Shooter Current Slave", () -> slave.getOutputCurrent());
```

Comment:
It looks like these values will not be logged after your change. I recommend achieving equivalent functionality by adding an `initSendable()` implementation which registers the properties and callling `SmartDashboard.putData(shooter)` in `RobotContainer` to ensure they get logged.